cookbook/running-sulu-on-heroku.rst

```diff
@@ -0,0 +1,33 @@
+Running Sulu on Heroku
+======================
+
+`Heroku`_ enables programmers to run their applications in the cloud, and makes
+it really easy to scale your application if it follows some rules. There is a 
+`Sulu cloud edition`_, which follows these rules.
+
+You will find a "Deploy to Heroku" button in the ``README.md`` file of the
+repository of the previously mentioned `Sulu cloud edition`_. This button leads
+to a Heroku page for deploying your very own Sulu installation.
+
+Give the application a name and choose between ``Europe`` and ``United States``
```

Comment:
I would say "Europe" and "United States" should be in normal (double) quotes. Double back-ticks should be reserved for things that would be parsed by something other than a human.